google/cloud/storage/doc/storage-main.dox

```diff
@@ -61,6 +61,20 @@ which should give you a taste of the Cloud Storage C++ client library API.
 - `CLOUD_STORAGE_TESTBENCH_ENDPOINT=...` override the default endpoint used by
   the library, intended for testing only.
 
+### Experimental
+
+- `GOOGLE_CLOUD_CPP_STORAGE_REST_CONFIG=...` configuration for the REST
+  protocol, currently only the `disable-xml` value has any effect. Sometimes
+  the application developer may want to test using an emulator that does not
+  support XML, while the library defaults to XML for some media operations.
+
+- `GOOGLE_CLOUD_CPP_STORAGE_GRPC_CONFIG...` used with
```

Comment:
Similar paragraphs include a `=` after the name.

Reply:
Fixed.